src/Validation/RequestValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+use WMDE\Fundraising\Entities\Request;
```

Comment:
Since one of the things I'll be doing today is integrating the 2.x dev branch of the fundraising store, I'll rename the Entity there to something less confusing.